packages/permission-controller/src/errors.test.ts

```diff
@@ -1,6 +1,26 @@
-import { EndowmentPermissionDoesNotExistError } from './errors';
+import {
+  CaveatMergeTypeMismatchError,
+  EndowmentPermissionDoesNotExistError,
+} from './errors';
 
 describe('error', () => {
+  describe('CaveatMergeTypeMismatchError', () => {
```

Comment:
Should we also be testing the other added errors?

Reply:
No, this is just for coverage purposes, and all other errors are covered.